### PR TITLE
Additional info to thirdparty doc

### LINKED ISSUE #739: Updates to ThirdPartyHosting.md
Add new information mentioning:
- malicious fork attack vector
- only use a third party you trust or for testing/getting started
- low balance, etc. 
- what third party host knows about users (count and username)

### PR DESCRIPTION
**Pull request by britttttk**
close #739

### CODE REVIEW

**Comment by britttttk** [comment]
this link https://electrondust.com/2017/11/25/setting-raspberry-pi-wifi-static-ip-raspbian-stretch-lite/ seems fine :/

File: docs/ThirdPartyHosting.md [hunk 1]
```diff
@@ -75,7 +82,37 @@ Specifically, third-party hosts should not enable the following policies without
...
 ## Can I enable the use of my Lightning Network node to others?
 
 Yes, you can enable users that have registered on your BTCPay Server instance to use your Lightning Network node.
 [See how](./FAQ/FAQ-LightningNetwork.md#how-many-users-can-use-lightning-network-in-btcpay)
+
+## What does the third-party host know about their users?
+Third party hosts can see the following:
+
+- Total number of users
+- The email username of those users
```
Comment (Zaxounette) [inline docs/ThirdPartyHosting.md:114]:
```suggestion
- The email and username of those users
```

**Review by Zaxounette — APPROVED** [review]

**Review by Zaxounette — APPROVED** [review]

File: docs/ThirdPartyHosting.md [hunk 2]
```diff
@@ -10,41 +10,48 @@ Third-party hosts play an important role in the ecosystem since they provide an
...
 
-### Security concerns
+## Concerns For Use
+
+### Security Concerns
 [Trusted third parties are security holes](https://nakamotoinstitute.org/trusted-third-parties/#selection-7.6-6.2). By relying on someone else to manage a server for you, you are potentially exposing yourself to a certain attack vector.
 
-The most significant attack vector when using a third-party host is that a malicious and technically skilled host can create a forked version of BTCPay Server and modify it so that it either spy on your transactions or replace your [xpubkey](https://en.bitcoin.it/wiki/Deterministic_wallet_tools#Risks_of_Sharing_an_Extended_Public_Key_.28xpub.29) with his. This means that future payments made to you may end up in a malicious party wallet.
+The most significant attack vector when using a third-party host is that a malicious and technically skilled host can create a forked version of BTCPay Server and modify it so that it either spy on your transactions or replace your [xpubkey](https://en.bitcoin.it/wiki/Deterministic_wallet_tools#Risks_of_Sharing_an_Extended_Public_Key_.28xpub.29) with his. This means that future payments made to you may end up in a malicious party wallet.
```
Comment (pavlenex) [inline docs/ThirdPartyHosting.md:48]: > or replace your [xpubkey]

IMO the bigger attact vector is using generate a wallet feature so third-party host generates a wallet for you and then steals funds, xpub is just a privacy concern and should be pushed to be used when using a third-party host.

File: docs/ThirdPartyHosting.md [hunk 3]
```diff
@@ -10,41 +10,48 @@ Third-party hosts play an important role in the ecosystem since they provide an
...
 
-The most significant attack vector when using a third-party host is that a malicious and technically skilled host can create a forked version of BTCPay Server and modify it so that it either spy on your transactions or replace your [xpubkey](https://en.bitcoin.it/wiki/Deterministic_wallet_tools#Risks_of_Sharing_an_Extended_Public_Key_.28xpub.29) with his. This means that future payments made to you may end up in a malicious party wallet.
+The most significant attack vector when using a third-party host is that a malicious and technically skilled host can create a forked version of BTCPay Server and modify it so that it either spy on your transactions or replace your [xpubkey](https://en.bitcoin.it/wiki/Deterministic_wallet_tools#Risks_of_Sharing_an_Extended_Public_Key_.28xpub.29) with his. This means that future payments made to you may end up in a malicious party wallet. 
+
+It is impossible to know for certain, if the third party host was using a malicious fork. If you do not personally know the third party host (and even so) it is best to minimize trust in your third party host by doing the following:
```
Comment (pavlenex) [inline docs/ThirdPartyHosting.md:50]: > If you do not personally know the third party host 

Perhaps just "if you don't trust"? Personally known seems a bit of an overstatement considering we live in a digital world :)

File: docs/ThirdPartyHosting.md [hunk 4]
```diff
@@ -75,7 +82,36 @@ Specifically, third-party hosts should not enable the following policies without
...
+* No [Server Settings](./Walkthrough.md#server-settings) access
+
+The limitations happen for technical reasons, mostly because these features require a user to run a full node to use them.
+
 ## Can I enable the use of my Lightning Network node to others?
 
 Yes, you can enable users that have registered on your BTCPay Server instance to use your Lightning Network node.
 [See how](./FAQ/FAQ-LightningNetwork.md#how-many-users-can-use-lightning-network-in-btcpay)
+
+## What does the third-party host know about their users?
```
Comment (pavlenex) [inline docs/ThirdPartyHosting.md:110]: Not sure how I'd feel about this, since if malicious they can know way beyond this, right?

**Review by pavlenex — APPROVED** [review]

**Comment by pavlenex** [comment]
Test are failing now because of:
```
 jq: error: Could not open file BTCPayServer/wwwroot/swagger/v1/*.json: No such file or directory
Error: Process completed with exit code 2.
```

Which is unrelated to this PR

**Pull request merged**